tests/transformers/helper.py

```diff
@@ -257,6 +257,33 @@ def load_feature_extraction_pipeline():
     return transformers.pipeline(model=model, tokenizer=tokenizer, task="feature-extraction")
 
 
+@prefetch
+@flaky()
+def load_peft_pipeline():
+    try:
+        import torch
+        from peft import PeftConfig, PeftModel
+    except ImportError:
+        # Do nothing if PEFT/Pytorch is not installed
+        return
+
+    peft_model_id = "ybelkada/opt-350m-lora"
```

Comment:
this model is quite big, can we use a small model like `bert-tiny`? https://huggingface.co/prajjwal1/bert-tiny, actually the best way IMO is to build a toy PyTorch model (need to subclass from `PretrainedModel` class) and convert it to PEFT.

Reply:
Sure just copied this one from random notebook, thanks for finding the better one! Yeah agree that we may want to create light-weight custom model, but it's a bit challenging as we have tons of other pipeline types.

Reply:
no worries, bert-tiny is just a few MBs, we should be good to go.